SparkPost/2web2ui
# SA-1721: Add filter link works for comparators table

### What Changed
 - Group by table adds filters for new filter structure

### How To Test
 - On hibana account with feature flag `allow_report_filters_v2`, go to `/signals/analytics`
 - Go to bottom group by table and select a filter
 - Go to group by table
 - Click link to add filter
 - Verify that link adds filter to the table

### To Do
